src/main/java/seedu/address/logic/parser/ClassDeleteStudentCommandParser.java

```diff
@@ -37,7 +37,7 @@ public ClassDeleteStudentCommand parse(String args) throws ParseException {
         ClassroomParserUtil.parseClassName(className);
         String moduleCode = argMultimap.getValue(PREFIX_MODULE_CODE).get().toUpperCase();
         ParserUtil.parseModuleCode(moduleCode);
-        String matricNo = ParserUtil.parseMatric(argMultimap.getValue(PREFIX_MATRIC).get());
+        String matricNo = ParserUtil.parseMatric(argMultimap.getValue(PREFIX_MATRIC).get()).toString(); // TBC
```

Comment:
Perhaps you can keep the `MatricNo` object and invoke its `#toString()` whenever you need it in `String` representation

Reply:
Just realised that this is for @zhicaizack to decide